orangegrove.rb

```diff
@@ -0,0 +1,85 @@
+# ORANGE TREE ------------------------------------------------------------------------------->
+class OrangeTree
+
```

Comment:
Try to use two spaces instead of tabs for your indentation. You can change your editor to use two spaces instead of a tab when you indent.